### DISCUSSION TITLE
Hidden pages on the left-side panel

### DISCUSSION THREAD

**Discussion by debbie-oyek** [GitbookIO/community#680] [Feature Requests]
Is there a way to keep hidden pages visible (but not published) on the left-side panel even after publishing? When a page is hidden, it disappears from view after the site is published. They will only come up when you enter the edit mode. In some cases, you might want to review or reference the site without having to go into edit mode, and if you don't go into edit mode for a while, you might forget that there are hidden pages there.  Thanks!

**Comment by addisonschultz** [comment]
Hi @debbie-oyek - we’ve just fixed this so you will now see hidden pages in the editor mode, without having to enter into a Change Request!

**Reply by ahcsco** [reply]
Hi, When will this fix be issued? + the publish site shows a drop-down button for hidden page.

https://github.com/user-attachments/assets/c8b169c7-9dbc-4384-9821-209c9021e7cd

**Reply by addisonschultz** [reply]
@ahcsco you can also hide page groups - which will hide the dropdown you have in the video! See https://docs.gitbook.com/creating-content/content-structure/page#visibility for more info.

If you hide a page group, the pages inside the group will inherit it’s visiblity

**Reply by ahcsco** [reply]
Hi, Thanks, but that's not what I'm after here. I want to hide one specific page under the group while keeping the others published. Could you say if the fix you say has been issued as I am still seeing the need to open a CR to see hidden pages, + if something can be done about this one too in video?

**Reply by addisonschultz** [reply]
Hi @ahcsco I don’t think I’m understanding what you’re after - the fix has been deployed already where you can now see a hidden page in the GitBook **editor**, regardless if you’re in a change request or not. Can you please share a larger video of what you’re experiencing if that’s not the case? 

In the published section - in your video there are no other pages in the published section that I can see - so I’m not sure what you are referring to as “ I want to hide one specific page under the group while keeping **the others** published.”?

**Reply by ahcsco** [reply]
OK, i'll clarify. 

In the video below, I hid a page, then merged the change, came back to the editor, but the page remains invisible without going back into CR. This contradicts the statement "the fix has been deployed already where you can now see a hidden page in the GitBook editor, regardless if you’re in a change request or not". 

https://github.com/user-attachments/assets/f82c98f0-62ff-4bc7-91c5-b7c24c4eb78e

On the Publish site (and this one is just UX, not a biggie if not fixed): if a page is set to hide, there would be no need to show a drop-down on its parent page when there's only one page under the parent.

https://github.com/user-attachments/assets/cb5afab9-31b8-4634-a081-9175abbc2875

**Reply by addisonschultz** [reply]
Hi @ahcsco - the first one is a bug! Can you try to hard refresh your browser to see if that fixes it? Does it appear after a short period of time? Or does it always stay invisible?

For the 2nd one - because the page group itself can have content on it, it has to be explicitly hidden as well (otherwise if we hid it by default if there was only one child, the top level page with content on it might also be hidden which would be undesirable)

**Reply by ahcsco** [reply]
Hi, the first one persists. But that's OK. I'll leave it here and unhide the page.
#2 can be ignored.

**Comment by debbie-oyek** [comment]
Super! Thanks Addison.
